# Define the goal

bool start_mapping
bool finish_control_mapping
bool check_map
bool move_goal
geometry_msgs/Pose2D goal

bool record_position
string position_key

---
# Define the result
bool finish

---
# Define a feedback message
string status